ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot

```diff
@@ -0,0 +1,152 @@
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Check Project Metrics Default Page Contents    ${PRJ_TITLE}
+
+Verify Distributed Workload status Default Page contents
+    [Tags]    RHOAIENG-4837
+    ...       Sanity    DistributedWorkloads
+    [Documentation]    Verifiy distributed workload status page default contents
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Wait until Element is Visible    xpath=//div[text()="Distributed workload resource metrics"]   timeout=20
+    Check Distributed Workload Status Page Contents
+
+Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Documentation]    Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Tags]    RHOAIENG-4837
+    ...       Tier1    DistributedWorkloads
+    Launch Dashboard    ocp_user_name=${TEST_USER_3.USERNAME}    ocp_user_pw=${TEST_USER_3.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER_3.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    ...    browser=${BROWSER.NAME}    browser_options=${BROWSER.OPTIONS}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE_NONADMIN}   description=${PRJ_DESCRIPTION}
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE_NONADMIN}
+    Wait until Element is Visible    xpath=//h4[text()="Configure the project queue"]   timeout=20
+    Page Should Contain Element     xpath=//div[text()="Configure the queue for this project, or select a different project."]
+    # setup Kueue resource for the created project
+    Setup Kueue Resources    ${PRJ_TITLE_NONADMIN}
+    Click Link    Distributed Workload Metrics
+    Select Distributed Workload Project By Name    ${PRJ_TITLE_NONADMIN}
+    Check Project Metrics Default Page Contents    ${PRJ_TITLE_NONADMIN}
+    Check Distributed Workload Status Page Contents
+    [Teardown]    Run Keywords
+    ...    Delete Data Science Project   ${PRJ_TITLE_NONADMIN}
+    ...    AND
+    ...    Wait Until Data Science Project Is Deleted  ${PRJ_TITLE_NONADMIN}
+    ...    AND
+    ...    Switch Browser    1
+
+
+*** Keywords ***
+Project Suite Setup
+    [Documentation]    Suite setup steps for testing Distributed workload Metrics UI
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    Launch Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ...    ${ODH_DASHBOARD_URL}    ${BROWSER.NAME}    ${BROWSER.OPTIONS}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Set Global Variable    ${project_created}    True
+    # setup Kueue resource for the created project
+    Setup Kueue Resources    ${PRJ_TITLE}
```

Comment:
Can you make sure that resources are also deleted after test execution?

Reply:
sure

Reply:
tested in PR check CI pipeline `rhods-ci-pr-test/2703`